client/login/magic-login/emailed-login-link-successfully.jsx

```diff
@@ -63,24 +45,34 @@ class EmailedLoginLinkSuccessfully extends Component {
 					waitForEmailAddress={ emailAddress }
 				/>
 
-				<h1 className="magic-login__form-header">{ translate( 'Check your email!' ) }</h1>
+				<h1 className="magic-login__form-header">{ translate( 'Check your email' ) }</h1>
 
 				<Card className="magic-login__form">
-					<p>{ preventWidows( line ) }</p>
+					<div className="magic-login__form-text">
+						<p>{ preventWidows( successMessage ) }</p>
+					</div>
 				</Card>
+				<div className="magic-login__emails-list">
+					<MagicLoginEmailWrapper emailAddress={ emailAddress } />
+				</div>
 
 				<div className="magic-login__footer">
-					<a
-						href={ login( {
-							isJetpack: this.props.isJetpackLogin,
-							isWhiteLogin: this.props.isWhiteLogin,
-							locale: this.props.locale,
-						} ) }
-						onClick={ this.onClickBackLink }
-					>
-						<Gridicon icon="arrow-left" size={ 18 } />
-						{ translate( 'Back to login' ) }
-					</a>
+					<p>
+						{ translate(
+							"Didn't get the email? You might want to double check if the email address is associated with your account,{{a}}or reset your password.{{/a}}",
```

Comment:
```suggestion
							"Didn't get the email? You might want to double check if the email address is associated with your account, {{a}}or reset your password.{{/a}}",
```
I think that there should be a space after the comma?

Reply:
Hi 👋 
The spacing is done using https://github.com/Automattic/wp-calypso/pull/84431/files#diff-f3418ecf5bbbe75b9b54d30784a467869f49ef4dd83f80c80c778809117d3193R193.

<img width="439" alt="image" src="https://github.com/Automattic/wp-calypso/assets/2653810/5d1af47f-f82f-4cd4-bd0a-eaff0ab5f7d3">

Reply:
It's a small issue since in a lot of translations there is no comma or the link is in a different place, but some translators might assume a typo and add a space after a comma anyway. Just for future consideration 🙂